Tools/Scripts/webkitdirs.pm

```diff
@@ -553,31 +553,13 @@ sub determineArchitecture
     $architecture = nativeArchitecture([]);
     if (isAppleCocoaWebKit() && $architecture eq "arm64") {
         determineXcodeSDK();
-        if ($xcodeSDK eq "macosx.internal") {
+        if ($xcodeSDK =~ /osx?\.internal$/) {
```

Comment:
Perhaps it's better to be explicit that `simulator` is not present in the identifier?